### ISSUE TITLE
subprocess.CREATE_NEW_PROCESS_GROUP only exists on Windows

### ISSUE THREAD

**Issue by haroldmeerwaldt** [stephanlphilips/core_tools#13]
Hi again @sldesnoo-Delft,

Our remote pipeline runs on Linux and fails on trying to import subprocess.CREATE_NEW_PROCESS_GROUP:

```python
src/lls/utils/core_tools/__init__.py:12: in <module>
    from core_tools.drivers.hardware.virtual_gate_matrix import VirtualGateMatrix, VirtualGateMatrixView
/usr/local/lib/python3.9/site-packages/core_tools/__init__.py:3: in <module>
    from core_tools.startup.launch_databrowser import launch_databrowser
/usr/local/lib/python3.9/site-packages/core_tools/startup/launch_databrowser.py:1: in <module>
    from core_tools.startup.app_launcher import launch_app
/usr/local/lib/python3.9/site-packages/core_tools/startup/app_launcher.py:6: in <module>
    from subprocess import Popen, CREATE_NEW_PROCESS_GROUP, PIPE, STDOUT
E   ImportError: cannot import name 'CREATE_NEW_PROCESS_GROUP' from 'subprocess' (/usr/local/lib/python3.9/subprocess.py)
```

Would it be possible to have the import be conditional on being on Windows, or import it locally inside `launch_app`?

Let me know if you'd like me to submit a PR.

Sincerely, Harold

**Comment by sldesnoo-Delft** [comment]
I've made the flag specific for Windows. Should be fixed in v1.4.2

**Comment by haroldmeerwaldt** [comment]
Nice! Thank you
